dist/plist-parse.js

```diff
@@ -229,7 +229,282 @@ function parsePlistXML (node) {
 }
 
 }).call(this)}).call(this,require("buffer").Buffer)
-},{"./xmldom/dom-parser":2,"buffer":7}],2:[function(require,module,exports){
+},{"@xmldom/xmldom":6,"buffer":9}],2:[function(require,module,exports){
```

Comment:
Not sure why all this changed, I just ran 'make' and this updated. Can revert if necessary.